.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+// For format details, see https://aka.ms/devcontainer.json.
+{
+  "name": "Sparrow Reference Web App Development Dockerfile",
+
+  // Sets the run context to one level up instead of the .devcontainer folder.
+  "context": "..",
+
+  // Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
+  "dockerFile": "../Dockerfile.dev",
+
+  // Set *default* container specific settings.json values on container create.
+  "settings": {},
+
+  // Add the IDs of extensions you want installed when the container is created.
+  "extensions": [
+    "esbenp.prettier-vscode",
+    "eamodio.gitlens",
```

Comment:
You can set this in your personal settings.json so as not to pollute the shared settings.

![image](https://user-images.githubusercontent.com/22032748/160171166-3660bf52-83e4-468c-9dfc-12296e5d6e61.png)


```json
"remote.containers.defaultExtensions": [
  "lfs.vscode-emacs-friendly",
  "streetsidesoftware.code-spell-checker",
  "eamodio.gitlens"
]
```